src/test/java/net/glowstone/constants/ParticleTest.java

```diff
@@ -43,17 +43,20 @@ public void testGetData() {
 
         switch (particle) {
             case ITEM_BREAK:
-                assertEquals("Wrong getData for " + particle, true, particle.getData() != null);
-                assertArrayEquals("Wrong extData for " + particle, new int[]{Material.STONE.getId(), 1}, GlowParticle.getData(particle, STONE));
+                assertEquals("Wrong data for " + particle, true, particle.getData() != null);
+                assertArrayEquals("Wrong extData for " + particle, new int[]{Material.STONE.getId(), 1}, GlowParticle.getExtData(particle, STONE));
```

Comment:
Do you think that extra data would be more readable vs extData? I'm not sure because it deviates from the actual variable name, but gives it it's friendly name.